### PR TITLE
[Docs] Updated emojis

### LINKED ISSUE #321: [Docs] Replace Github Emojis with their image equivalents
In `docs/index.md` there are three emojis (for example `:hammer:`). They are shown correctly in the preview (for example like this: :hammer: ) but the HTML rendering at https://googlechrome.github.io/rendertron doesn't show them correctly.

1. Look at the preview in https://github.com/GoogleChrome/rendertron/blob/master/docs/index.md and right click an emoji to get the image address
2. Edit `docs/index.md` to use the image instead of the `:hammer:` etc.

To do (2) you will need to replace the emojis (for example `:hammer:`) with the corresponding image (for example ``)

### PR DESCRIPTION
**Pull request by mfaheemakhtar**
There are a couple of issues here.

1. The `img` tag is being wrapped in an `a` tag. One way is to set `href` to `#` (which I've done).

2. When using the actual emoji, it is being replaced by an g-emoji element, which has its own styles plus it sets the height and width to '1rem'.

TLDR: The position and size is not absolutely same as the emoji.

Resolves #321

### CODE REVIEW

**Comment by AVGP** [comment]
Hey @mfaheemakhtar - no worries, this isn't about being pixel-perfect!
This looks pretty good to me - thank you very much for your contribution and the work you put into this 👍 

Cheers 🍻
